addons/breathing/stringtable.xml

```diff
@@ -1357,246 +1364,287 @@
             <French>Remplissage du réservoir à oxygène portable</French>
             <Korean>휴대용 산소 탱크 재충전 중</Korean>
             <Japanese>携帯用酸素ボンベを補充中</Japanese>
+            <German>Befülle tragbaren Sauerstofftank</German>
         </Key>
         <Key ID="STR_KAT_Breathing_RefillPortableOxygenTank_Complete">
             <English>Oxygen Tank Refill Complete</English>
             <French>Remplissage du réservoir à oxygène terminé</French>
             <Korean>휴대용 산소 탱크 재충전 완료</Korean>
             <Japanese>酸素ボンベの補充が完了しました</Japanese>
+            <German>Sauerstofftank Befüllung abgeschlossen</German>
         </Key>
         <Key ID="STR_KAT_Breathing_RefillPortableOxygenTank_Cancel">
             <English>Oxygen Tank Refill Cancelled</English>
             <French>Remplissage du réservoir à oxygène annulé</French>
             <Korean>휴대용 산소 탱크 재충전 취소</Korean>
             <Japanese>酸素ボンベの補充が中断されました</Japanese>
+            <German>Sauerstofftank Befüllung abgebrochen</German>
         </Key>
         <Key ID="STR_KAT_Breathing_RefillPortableOxygenTanks">
             <English>Refill Portable Oxygen Tank</English>
             <French>Remplir le réservoir à oxygène portable</French>
             <Korean>휴대용 산소 탱크 재충전</Korean>
             <Japanese>携帯用酸素ボンベ を補充する</Japanese>
+            <German>Nachfüllen des tragbaren Sauerstofftanks</German>
         </Key>
         <Key ID="STR_KAT_Breathing_RefillPortableOxygenTank_150">
             <English>Refill Portable Oxygen Tank (150L)</English>
             <French>Remplir le réservoir à oxygène portable (150L)</French>
             <Korean>휴대용 산소 탱크 (150L) 재충전</Korean>
             <Japanese>携帯用酸素ボンベ (150L) を補充</Japanese>
+            <German>Nachfüllen des tragbaren Sauerstofftanks (150L)</German>
         </Key>
         <Key ID="STR_KAT_Breathing_RefillPortableOxygenTank_300">
             <English>Refill Portable Oxygen Tank (300L)</English>
             <French>Remplir le réservoir à oxygène portable (300L)</French>
             <Korean>휴대용 산소 탱크 (300L) 재충전</Korean>
             <Japanese>携帯用酸素ボンベ (150L) を補充</Japanese>
+            <German>Nachfüllen des tragbaren Sauerstofftanks (300L)</German>
         </Key>
         <Key ID="STR_KAT_Breathing_BVM_Display">
             <English>BVM</English>
             <French>BVM</French>
             <Korean>인공호흡기</Korean>
             <Japanese>バッグバルブマスク (BVM)</Japanese>
+            <German>BVM</German>
         </Key>
         <Key ID="STR_KAT_Breathing_BVM_Desc_Short">
             <English>Bag-Valve-Mask, used to assist with breathing of unconscious patients</English>
             <French>Ballon-masque, utilisé pour assister la respiration de patients inconscients</French>
             <Korean>의식불명인 환자의 호흡을 보조하는 데 사용되는 인공호흡기</Korean>
             <Japanese>意識を失った患者の呼吸補助に使用</Japanese>
+            <German>Die Beutelventilmaske wird benutzt um das Atmen von bewustlosen Personen zu erleichtern</German>
```

Comment:
Die Beutelventilmaske wird benutzt um das Atmen von bewustlosen Personen zu erleichtern
> means you're breathing uncon patients instead of air.

Die Beutelventilmaske wird benutzt um die Atmung von bewustlosen Patienten zu unterstützen